catroid/src/org/catrobat/catroid/io/SoundManager.java

```diff
@@ -61,6 +61,35 @@ public synchronized void playSoundFile(String pathToSoundfile) {
 		}
 	}
 
+	public synchronized void playSoundFileAndWait(String pathToSoundfile) {
```

Comment:
this method is unnecessary, since it's the same as the existing ``public synchronized void playSoundFile(String pathToSoundfile)``